sdk/attestation/Azure.Security.Attestation/README.md

```diff
@@ -263,9 +270,9 @@ See [CONTRIBUTING.md][contributing] for details on building, testing, and contri
 [attestation_response]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.attestationresponse-1
 [attestation_response_token]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.attestationresponse-1.token
 [attestation_response_value]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.attestationresponse-1.value
-[attestation_policy_result]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.models.policyresult
-[attestation_policy_result_signer]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.models.policyresult.policysigner
-[attestation_policy_result_token_hash]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.models.policyresult.policytokenhash
+[attestation_policy_result]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.policyresult
+[attestation_policy_result_signer]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.policyresult.policysigner
+[attestation_policy_result_token_hash]: https://docs.microsoft.com/dotnet/api/azure.security.attestation.policyresult.policytokenhash
```

Comment:
These links are now broken. @LarryOsterman

Reply:
Please file an issue on this so this doesn't get lost.